### PR TITLE
Fix license violation of git-version-bump gem

### PR DESCRIPTION
**Pull request by lksv**
This fixes issue #125 as described there.

### CODE REVIEW

**Comment by chneukirchen** [comment]
This breaks a public API... but only if anyone even used it.

**Comment by lksv** [comment]
Unfortunately yes, it breaks. There is no other easy way:(  ...and I understand that increasing major version number is not usually done every day.

**Comment by stereobooster** [comment]
Additionally we do not need it at runtime.

```
bundle exec derailed bundle:objects | head -n 100
Measuring objects created by gems in groups [:default, "production"]
Total allocated: 177429400 bytes (1309385 objects)
Total retained:  55128450 bytes (189659 objects)

allocated memory by gem
-----------------------------------
  86323706  activesupport-3.2.22.4
   9460678  git-version-bump-0.15.1
   8287164  erubis-2.7.0
```

Can I help you to get this merged?

**Comment by libc** [comment]
I think it's better to remove it. Because right now including `rack-contrib` via bundler fails unless you have `git-version-bump` installed. It's easy to reproduce if you have no gems installed, and then try to run `bundle install` with `gem 'rack-contrib', github: 'rack/rack-contrib'`.

It fails because when evaluating `gemspec` the version does not match when `git-version-bump` is installed and when it's not installed.

PR should be easy, but this project seems to be unmaintained.

File: lib/rack/contrib.rb [hunk 1]
```diff
@@ -3,9 +3,6 @@
 
 module Rack
   module Contrib
-    def self.release
-      GVB.version
```
Comment (mpalmer) [inline lib/rack/contrib.rb]: This shouldn't be removed, instead it should be modified to use whatever mechanism you propose to use for maintaining version information.

**Comment by mpalmer** [comment]
Superceded by #114.

**Pull request closed without merging**